C/Coin-OR/Ipopt/build_tarballs.jl

```diff
@@ -73,5 +74,11 @@ build_tarballs(
     products,
     dependencies;
     preferred_gcc_version = gcc_version,
-    julia_compat = "1.6"
+    julia_compat = "1.6",
+    init_block = """
+    @static if VERSION < v"1.7.0-DEV.641"
+            ccall((:lbt_forward, libblastrampoline), Int32, (Cstring, Int32, Int32),
+                  OpenBLAS32_jll.libopenblas_path , 1, 0)
+        end
+    """
```

Comment:
To be honest, I'm not 100% sure it's a good idea to put this here.  For example, you had to do the same in MUMPS: doing it multiple times in multiple places is not OK.  We do this
https://github.com/JuliaPackaging/Yggdrasil/blob/1e66c6f8ddea9885382efb393b2cd33db2ce42dc/L/libblastrampoline/build_tarballs.jl#L38-L43
in libblastrampoline directly for Julia v1.6, but there we are sure we have an ILP64 BLAS.

Well, I guess we'll see how this pans out

Reply:
Here we're sure we have an LP BLAS because we install OpenBLAS32, right? And if one redirects what library LBT "points to", all packages that depend on libblastrampoline_jll will subsequently use that other BLAS in unison, right?